core/src/test/java/org/testcontainers/utility/PrefixingImageNameSubstitutorTest.java

```diff
@@ -0,0 +1,75 @@
+package org.testcontainers.utility;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.rnorth.visibleassertions.VisibleAssertions.assertEquals;
+import static org.testcontainers.utility.PrefixingImageNameSubstitutor.PROPERTY_KEY;
+
+public class PrefixingImageNameSubstitutorTest {
```

Comment:
we should also add a test for images from custom registries (e.g. `mcr.microsoft.com/mssql/server`), to define the behaviour

Reply:
Removed from this PR